contrib/cirrus/test.sh

```diff
@@ -34,7 +34,7 @@ then
             export GITVALIDATE_TIP="$CIRRUS_CHANGE_IN_REPO"
```

Comment:
[lines not shown in github diff] Please remove `ooe.sh` from lines above

Reply:
`ooe.sh` is now part of the VM image, put there at build time from containers/automation repo.